fda-model/paireddata/StepwisePairedData.cs

```diff
@@ -93,11 +93,11 @@ public IPairedData multiply(IPairedData g)
 				        newYvals.Add(belowFragilityCurveValue);
 			        } else {
 				        //create a point on the curve just below the bottom of the levee at damage zero.
-				        newXvals.Add(bottom-.000000000001);
+				        newXvals.Add(bottom-.001);
```

Comment:
why do you decrease the value to .001?

Reply:
in Fragility curve units, those are feet. I figured a 1/1000th of a foot buffer was probably enough, and made it easier to write tests against it. Too hard to count how many zeroes were there when predicting the answers.

Reply:
Weird that it's in this pull request though. I may have gotten mixed up there. It is definitely unrelated to the stats bug.